poetry.lock

```diff
@@ -33,11 +33,11 @@ frozenlist = ">=1.1.0"
 
 [[package]]
 name = "asgiref"
-version = "3.4.1"
```

Comment:
Did all these updates come up when you installed pysolc? Or did you run an update command?